site/src/examples/stepper-input/TextAlignment.tsx

```diff
@@ -0,0 +1,19 @@
+import { FormField, FormFieldLabel, StackLayout } from "@salt-ds/core";
+import { StepperInput } from "@salt-ds/lab";
+
+export const TextAlignments = () => (
```

Comment:
```suggestion
export const TextAlignment = () => (
```
This name means the example doesn't show